Doc/conf.py

```diff
@@ -286,7 +286,7 @@
     print("It may be removed in the future\n")
 
 # Short title used e.g. for <title> HTML tags.
-html_short_title = f'{release} Documentation'
+html_short_title = f'Python {".".join(release.split(".")[:2])} Documentation'
```

Comment:
I'm not convinced we need to make this change, but we already have x.y: https://github.com/Yzi-Li/cpython/blob/f3a515984430c932989adec7378d4e4fc130cdb7/Doc/conf.py#L77 

```suggestion
html_short_title = f'Python {version} Documentation'
```